webpack/assets/javascripts/react_app/routes/common/PageLayout/PageLayout.js

```diff
@@ -20,11 +20,13 @@ const PageLayout = ({
   beforeToolbarComponent,
   isLoading,
   children,
+  className,
+  title,
 }) => (
-  <div id="main">
+  <div id="main" className={className}>
     <div id="react-content">
       <Head>
-        <title>{header}</title>
+        <title>{title || header}</title>
```

Comment:
Why do we need both `title` and `header` props?

Reply:
atm, the default title is the header of the page,
we should allow an option to edit the title as some plugins already do,
so for example the title would be: `My App - Architectures`, and the page header can remain `Architectures`

Reply:
But wouldn't then these plugins need to also know the header (e.g. `Architectures` in your example) ? Would it be better in such a case to use `titleTemplate` attribute in Helmet?

Reply:
Agreed, thanks @tbrisker

Reply:
passing now `headProps` into the <Head /> component to cover more options